src/components/hooks/use-visibility.tsx

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react'
+import { TUseBreakpoints } from './use-breakpoints'
+import { DerivStore } from 'store'
+
+// HINT: We will add more values to the hook in the future,
+// so for the type handling for now I'll just alias the type
+export type TVisiblity = TUseBreakpoints
+
+const useVisibility = (): TVisiblity => {
+    const { breakpoints } = useContext(DerivStore)
+
+    return breakpoints
```

Comment:
@michio-rework I got a little confused here, Maybe it’s because of the namings. I see you will be adding more stuff to this hook later but couldn’t we just use `useBreakpoints` here instead of `useContext`? It returns the same thing, Right 🤔

Reply:
another thing is, we will create a separate `Context` for the screen sizes. DerivStore is handling too much.